### DISCUSSION TITLE
GitHub Actions / Discussions

### DISCUSSION THREAD

**Discussion by jtr13** [jtr13/cc22spring#21] [General]
It's working now! I forgot to mention that the collaborator who merged the pull request will a receive an email notification if GitHub Actions fails, so you can rely on that to know if there's a problem. If you want to be notified either way, uncheck "Send notifications for failed workflows only" in the Actions section of https://github.com/settings/notifications

Also, we can use this Discusssions section for questions/comments that aren't really "Issues" -- we'll reserve "Issues" for problems that need to be fixed.

@codingellena @howardchang1 @tu-yujie @ytyky

**Comment by tu-yujie** [comment]
It seems we have a missing package "FactoMineR", should this kind of problems go into the Issue?

**Comment by jtr13** [comment]
Yes, though no need this time as I'm working on it!

**Comment by jtr13** [comment]
Just checking in to see what the plan is. It would be great to get this done so students can use the contributions as they work on their final projects. Please let me know if you need any help.

**Reply by tu-yujie** [reply]
Yes, we have discussed about it and we plan to finish this by the end of this week

**Reply by jtr13** [reply]
Fantastic, it's looking great!

**Comment by tu-yujie** [comment]
Just trying to make sure. If the pull request comes from the main branch, we should simply reject the pull request, right?

**Comment by jtr13** [comment]
Correct, thanks for checking.
